src/stores/Frame/Region/RegionStore.ts

```diff
@@ -198,6 +198,10 @@ export class RegionStore {
         }
     }
 
+    /**
+     * Getter for the sizes of regions and annotations
+     * @returns A Point2D object with x and y size components.
```

Comment:
I suggest we revise it to "The x and y size components." (We don't need to include types.)